code/scripting/lua.cpp

```diff
@@ -271,6 +271,11 @@ void script_state::OutputLuaMeta(FILE *fp)
 	ade_table_entry *ate;
 	fputs("<dl>\n", fp);
 
+	//***Version and copyright info
+	fprintf(fp, "<dd>%s</dd>\n", LUA_RELEASE);
+	fprintf(fp, "<dd>%s</dd>\n", LUA_COPYRIGHT);
```

Comment:
Do we need the Lua copyright in the scripting documentation? If it's included like this it might suggest that the documentation is covered by this copyright which is not the case. Maybe it could be clarified by using something like `<dd>Lua Copyright: %s</dd>`.